heim-process/src/sys/windows/process/io_counters.rs

```diff
@@ -0,0 +1,74 @@
+use heim_common::units::{information, Information};
+use std::fmt;
+use winapi::um::winnt;
+/// Process IO statistics.
+/// For additional information, see [IO_COUNTERS] documentation.
+///
+/// [IO_COUNTERS]: https://docs.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-io_counters
+#[derive(Default)]
```

Comment:
Deriving `Default` seems unnecessary, this struct is not instantiated anywhere in a blank state